### PR TITLE
Trans to vietnamese

### PR DESCRIPTION
**Pull request by LeoTeam**
We translated 4 posts to Vietnamese.
https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/ https://www.ruby-lang.org/en/news/2015/04/13/ruby-2-2-2-released/
https://www.ruby-lang.org/en/news/2015/04/13/ruby-2-1-6-released/
https://www.ruby-lang.org/en/news/2015/04/13/ruby-2-0-0-p645-released/

### CODE REVIEW

File: vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+translator: "My Mai"
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Chúng tôi rất vui mừng thông báo việc phát hành phiên bản Ruby 2.0.0-p645.
+
+Lần phát hành này bao gồm việc sửa lỗi bảo mật cho tiện ích OpenSSL. Để biết thêm chi tiết xin vui lòng xem chủ đề dưới đây.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: The title should use Vietnamese: `Xác nhận Hostname Ruby OpenSSL` and should link to trasnlation: `https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`:

```
* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

:smile:

File: vi/news/_posts/2015-04-13-ruby-2-1-6-released.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Phiên bản Ruby 2.1.6 đã được phát hành.
+
+Phiên bản phát hành lần này bao gồm việc sửa chữa bảo mật cho OpenSSL extension.
+Xem các chủ đề bên dưới để biết thêm chi tiết.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline vi/news/_posts/2015-04-13-ruby-2-1-6-released.md]: The title should use Vietnamese: `Xác nhận Hostname Ruby OpenSSL` and should link to trasnlation: `https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`:

```
* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

:smile:

File: vi/news/_posts/2015-04-13-ruby-2-2-2-released.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+translator: "Dôn Đinh"
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Chúng tôi vui mừng thông báo về việc phát hành Ruby 2.2.2. Đây là một phiên bản nhỏ của seri 2.2.
+
+Phiên bản này đã sửa chữa lỗi bảo mật do lỗ hổng xác minh cho một hostname của OpenSSL extension.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```
Comment (JuanitoFatas) [inline vi/news/_posts/2015-04-13-ruby-2-2-2-released.md]: The title should use Vietnamese: `Xác nhận Hostname Ruby OpenSSL` and should link to trasnlation: `https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`:

```
* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

:smile:

File: vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+Tất cả mọi người đang hoạt động trên các phiên bản bị ảnh hưởng này nên nâng cấp ngay.
+
+## Các phiên bản bị ảnh hưởng
+
+* Tất cả phiên bản đầu tiên của Ruby 2.0 đến bản vá 2.0.0 p645
+* Tất cả phiên bản đầu tiên của Ruby 2.1 đến Ruby 2.1.6
+* Tất cả phiên bản đầu tiên của Ruby 2.2 đến Ruby 2.2.2
+* Sửa đổi đầu tiên 50292
+
+## Credits
```
Comment (JuanitoFatas) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: Cảm ơn?

Comment (LeoTeam) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: If you mean 'Credits' is 'Cảm ơn' then it's wrong. I'll update this.

Comment (JuanitoFatas) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: :+1: I was stealing this translation from previous post :stuck_out_tongue: https://www.ruby-lang.org/vi/news/2014/11/13/rexml-dos-cve-2014-8090. Thanks for clarifying!

Comment (mymai91) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: I'll show you :) Please take a look
vietgle_tra_t _-_d nh_nghia_c a_t __credit__trong_t _di n_l c_vi t

Comment (JuanitoFatas) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: Thanks My Mai! Learn some good vietnamese today :blush:

Comment (mymai91) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: (y)

File: vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Chúng tôi rất vui mừng thông báo việc phát hành phiên bản Ruby 2.0.0-p645.
+
+Lần phát hành này bao gồm việc sửa lỗi bảo mật cho tiện ích OpenSSL. Để biết thêm chi tiết xin vui lòng xem chủ đề dưới đây.
+
+* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
+
+Hiện tại, Ruby 2.0.0 đang ở giai đoạn bảo trì an ninh, cho đến ngày 24 tháng 2 năm 2016.
+Sau ngày đó, việc bảo trì của Ruby 2.0.0 sẽ được kết thúc.
+We recommend you start planning migration to newer versions of Ruby, such as 2.1 or 2.2.
```
Comment (nguyenducgiang) [inline vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: Dòng này quên chwabor đi này.

File: vi/news/_posts/2015-04-13-ruby-2-1-6-released.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+title: "Phát hành Ruby 2.1.6"
+author: "usa"
+translator: "My Mai"
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Phiên bản Ruby 2.1.6 đã được phát hành.
+
+Phiên bản phát hành lần này bao gồm việc sửa chữa bảo mật cho OpenSSL extension.
```
Comment (nguyenducgiang) [inline vi/news/_posts/2015-04-13-ruby-2-1-6-released.md]: ở đây để `OpenSSL extension`, nhưng ở bài trên lại là `tiện ích OpenSSL`, nên dịch thống nhất một kiểu.

File: vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+Lỗ hỗng này đã được xác định trong [CVE-2015-1855][CVE-2015-1855].
+
+Đó là lý do các bạn nên nâng cấp Ruby.
+
+## Chi tiết
+
+Sau khi xem qua [RFC 6125][RFC-6125] và [RFC 5280][RFC-5280], chúng tôi đã tìm thấy nhiều vi phạm của các hostname tương ứng, đặc biệt là xác nhận các ký tự đại diện.
+
+Hiện tại, tiện ích OpenSSL của Rubys sẽ cung cấp một chuỗi dựa trên các thuộc toán tương ứng, nghiêm ngặt dựa theo các khuyến cáo của các RFC. Đặc biệt, việc kết hợp của nhiều hơn một ký tự đại diện cho mỗi vấn đề/SAN không còn được cho phép. Đồng thời, bây giờ việc so sánh các giá trị sẽ không còn nhạy cảm.
```
Comment (nguyenducgiang) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: *Ruby không phải Rubys

Comment (nguyenducgiang) [inline vi/news/_posts/2015-04-13-ruby-openssl-hostname-matching-vulnerability.md]: "bây giờ việc so sánh các giá trị sẽ không còn nhạy cảm" câu này hơi tối nghĩa so với từ gốc case-insensitive. đề nghị các bạn improve.

File: vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md (hunk 5 above)
Comment (joneslee85) [inline vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: Please remove this line

File: vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+Chúng tôi đề nghị các bạn nên bắt đầu kế hoạch chuyển đến các phiên bản mới hơn của Ruby chẳng hạn như 2.1 hoặc 2.2.
+
+Xem  [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_0_0_645/ChangeLog) để biết thêm chi tiết.
```
Comment (joneslee85) [inline vi/news/_posts/2015-04-13-ruby-2-0-0-p645-released.md]: please remove 1 space after `Xem`

**Comment by stomar** [comment]
@joneslee85 Please pay attention that PRs do not merge other branches and that commit messages conform to our conventions.

**Pull request merged**